ironfish/src/mining/pool.ts

```diff
@@ -134,6 +137,15 @@ export class MiningPool {
 
     this.logger.info('Connecting to node...')
     this.rpc.onClose.on(this.onDisconnectRpc)
+
+    const statusInterval = this.config.get('poolStatusNotificationInterval') ?? 0
```

Comment:
It should produce a warning about this null coelesence right? Since the config is typed as a number?

Reply:
I think that I had put this here because of a warning 🧐 , but it's not needed